src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/Drawing/DrawingContent/DrawingContent.tsx

```diff
@@ -55,13 +60,26 @@ export const DrawingContent = ({ name }: DrawingContentProps) => {
   ];
 
   return (
-    <Uploads
-      uploads={uploads}
-      wrapperStyles={{
-        mt: theme.spacing(2),
-        ml: theme.spacing(-4.8),
-        justifyContent: 'flex-start',
-      }}
-    />
+    <Box>
+      <Uploads
+        uploads={uploads}
+        wrapperStyles={{
+          mt: theme.spacing(2),
+          ml: theme.spacing(-4.8),
```

Comment:
Lets merge into one property
```suggestion
         m: theme.spacing(2, 0, 0, -4.8),
```